src/main/java/view25d/view/TurtleViewOptions.java

```diff
@@ -31,11 +35,34 @@ public TurtleViewOptions( TurtleView parent, boolean co, boolean sz, boolean sh,
 		shape.addChangeListener(this);
 		pcolor.addChangeListener(this);
 
-		this.add( new JLabel("View Options: ") );
+		this.add( new JLabel("  View Options:") );
 		this.add(color);
 		this.add(size);
 		this.add(shape);
 		this.add(pcolor);
+
+		threeDButton = new JRadioButton("ThreeD");
+		threeDButton.setActionCommand("threeD");
+		if (threeD) { threeDButton.setSelected(true); }
+
+		twoDButton = new JRadioButton("TwoD");
+		twoDButton.setActionCommand("TwoD");
+		if (twoD) { twoDButton.setSelected(true); }
+
+		ButtonGroup group2 = new ButtonGroup();
+		group2.add(threeDButton);
+		group2.add(twoDButton);
+
+		//Register a listener (myself) for the radio buttons.
```

Comment:
I don't think this comment adds much that the code itself doesn't say, but if you want to leave it in it should be signed and dated.